azure-pipelines.sds.qa-release.yml

```diff
@@ -0,0 +1,149 @@
+resources:
+  repositories:
+    - repository: azTemplates
+      type: github
+      name: hmcts/azure-devops-templates
+      ref: refs/heads/azure-dns-updates
+      endpoint: hmcts
+
+parameters:
+  - name: env
+    type: string
+    default: dev
+    values:
+      - dev
+
+  - name: qaenv
+    type: string
+    default: demo
+    values:
+      - demo
+
+  - name: git_repo_name
+    type: string
+    default: bookings-api
+
+trigger:
+  branches:
+    include:
+      - refs/tags/qa*
+
+variables:
+  - group: aks-cluster-info
+  - template: variables/shared.yaml
+    parameters:
+      env: ${{ parameters.env }}
+      git_repo_name: ${{ parameters.git_repo_name }}
+      cluster: $(cluster)
+  - name: prTagName
+    value: 'pr-379'
+  - name: dnsRecordName
+    value: ${{ parameters.git_repo_name }}-${{ variables.prTagName }}
+
+stages:
+  - stage: Uninstall
```

Comment:
Does the uninstall not only happen on PR close?

Reply:
this uninstalls from dev when it is tagged QA, we could remove it on PR close only but it might get messy